src/gdcdictionary/schemas/pathology_detail.yaml

```diff
@@ -982,13 +1080,6 @@ properties:
         - Base
         - Middle
 
-  tumor_measurement_method:
-    $ref:
-      - "_terms.yaml#/tumor_measurement_method/common"
-    enum:
-      - Echographic
-      - Pathologic
-
```

Comment:
Are these deprecated?
or just removed outright ?

Reply:
and what about it in `_terms.yaml`?

Reply:
We are removing these outright- they were not used and are redundant with another property.